src/+otp/+pendulum/PendulumProblem.m

```diff
@@ -0,0 +1,50 @@
+classdef PendulumProblem < otp.Problem
+    % PendulumProblem A two-variable model for the trajectory of a pendulum
+    %
+    %
+    %   This system can be reduced to the following two equations
+    %
+    %   theta' = omega
+    %   omega' = -g/l * sin(theta)
+    %
+    %   where g is the gravitational constant, l is the length of
+    %   the pendulum, and theta is the angle between the pendulum and
+    %   y-axis.
+     
+    methods
```

Comment:
Before we merge, we have to have a movie for the pendulum. This is one of the more perfect problems for this.